cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperatorTest.java

```diff
@@ -1345,35 +1336,26 @@ public void testReconcile(Params params, VertxTestContext context) {
 
         // providing the list of ALL StatefulSets for all the Kafka clusters
         Labels newLabels = Labels.forStrimziKind(Kafka.RESOURCE_KIND);
-        when(mockKsOps.list(eq(kafkaNamespace), eq(newLabels))).thenReturn(
-                asList(KafkaCluster.fromCrd(Reconciliation.DUMMY_RECONCILIATION, bar, VERSIONS).generateStatefulSet(openShift, null, null))
+        when(mockKsOps.list(eq("*"), eq(newLabels))).thenReturn(
```

Comment:
Why did you change `kafkaNamespace` ->` "*"`? Maybe a `wildcardNamespace` variable would be better?

Reply:
I don't think I did, I think this is something I rebased in, I will definitely check it out.

Reply:
Okay, that's odd, that was me, somehow, but really not sure how. Fixing it up.

Reply:
Resolving because ultimately realised I'd switched test names or bodies... ...somehow. I'm assuming while rebasing. Should look a lot better now.